cluster/plan.go

```diff
@@ -328,12 +336,14 @@ func (c *Cluster) BuildKubeAPIProcess(host *hosts.Host, serviceOptions v3.Kubern
 	}
 	registryAuthConfig, _, _ := docker.GetImageRegistryConfig(c.Services.KubeAPI.Image, c.PrivateRegistriesMap)
 
+	Env = append(Env, c.Services.KubeAPI.ExtraEnv...)
```

Comment:
I'm assuming order doesn't matter here (since we are reordering the environment variable putting `extraEnv` at the end instead of the beginning), correct?

Reply:
yep should be ok, upgrade check uses `sliceEqualsIgnoreOrder` under the hood https://github.com/rancher/rke/blob/1951add95d62585b8e21e191aea5ef0de516343f/docker/docker.go#L583 

will let Jack double check and confirm tho

Reply:
It is correct. The order here does not matter.  I wrote it this way at first because I planned to modify the getUniqStringList function to split each item into (key, value) pair and keep only the first occurrence of the key, but then I realized that it was unnecessary.